lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -113,6 +113,18 @@ bool PatchInOutImportExport::runOnModule(Module &module) {
   if (m_hasTs || (m_hasGs && (m_pipelineState->isGsOnChip() || m_gfxIp.major >= 9)))
     m_lds = Patch::getLdsVariable(m_pipelineState, m_module);
 
+  // Set buffer formats based on specific GFX
+  static const std::array<std::array<unsigned, 4>, 2> BufferFormats = {
+      {{
+           ((BUF_NUM_FORMAT_FLOAT << 4) | (BUF_DATA_FORMAT_32)),
```

Comment:
You don't need any of these parentheses.

Reply:
Fixed.